RobotCode/snobot2018/build.gradle

```diff
@@ -87,9 +87,10 @@ repositories {
 sourceSets {
     main {
         java {
-            srcDirs = ["src", "../SnobotRobotLibs/src"]
-            println new File("../SnobotRobotLibs/src").getAbsolutePath()
-            println new File("../SnobotRobotLibs/src").exists()
+            srcDirs = ["src", 
```

Comment:
I moved the stolen code into its own folder, so it would be easier to share with the standalone GUI